webapp/app/forms/steps/eligibility_steps.py

```diff
@@ -189,7 +189,10 @@ def __init__(self, **kwargs):
             **kwargs)
 
     def _main_handle(self, stored_data):
-        stored_data = super()._main_handle(stored_data)
+        super()._main_handle(stored_data)
```

Comment:
I know, that we do not need it here. However, imo we could still set it here as always just to ensure that we always do it like that